nebula-components/components/_btn.scss

```diff
@@ -44,6 +44,15 @@
     @include btn-color($nb-button-beta-color);
   }
 
+  .#{$nb-namespace}c-btn-icon {
+    position: absolute;
+    top: 0;
+    right: 0;
+    padding: (($nb-text-input-padding * 2) - 0.05);
```

Comment:
We should make a general rule that if we have to use a magic number we should either
a) Put it into a descriptive variable OR
b) Put a comment explaining why it's there / what it's doing.